docs/reference/dto/factory/index.rst

```diff
@@ -0,0 +1,12 @@
+dto.factory
+===========
+
+.. automodule:: litestar.dto.factory
```

Comment:
It has an automodule but is also the index for submodules?

Reply:
Well yeh, but I'm happy to change it.. the automodule brings in the things that are imported into the `dto.factory` namespace, and the TOC is for modules within. I'm still experimenting/finding my way around sphinx/autodoc, so happy to reorganize.